docs/security.rst

```diff
@@ -1,11 +1,16 @@
 Report a security issue
 =======================
 
+..
+  ATTENTION: For GitHub UI we also have /SECURITY.md file, which is a Markdown conversion of this file.
+  When making changes to this make, make sure to also update /SECURITY.md.
+
 We take security very seriously.
 Your input and feedback on our security is always appreciated.
 
-You can send urgent or sensitive reports directly to security@twistedmatrix.com
+You can send urgent or sensitive reports via `GitHub Security Advisory <https://github.com/twisted/twisted/security/advisories/new>`_.
```

Comment:
I think that we should also add a big warning at the start of this document, to mention that any security PR should be done via private repos using the GitHub functionalty.


Otherwise, by creating a normal PR the bug is disclosed before the fix being ready :)